core/src/com/projectkorra/projectkorra/firebending/FireShield.java

```diff
@@ -147,6 +149,13 @@ public void progress() {
 					entity.remove();
 				}
 			}
+			for (final Block block : GeneralMethods.getBlocksAroundPoint(this.location, this.shieldRadius)) {
+				if (ThreadLocalRandom.current().nextInt(5) == 0) {
+					dryWetBlocks(block, this, true);
+				} else {
+					dryWetBlocks(block, this);
+				}
```

Comment:
```suggestion
				dryWetBlocks(block, this, ThreadLocalRandom.current().nextInt(5) == 0);
```
Might also be worth saving the reference to current somewhere

Reply:
I'm not sure that's ideal. It defeats the whole purpose of calling a new random seed.

Reply:
Oh no, nevermind, I see what you're doing. Yeah that looks good.

Reply:
- Addressed in commit https://github.com/ProjectKorra/ProjectKorra/pull/1329/commits/5d2348b71b88876c1344d3b80052c7f95d17adad.